babel/src/main/java/org/apache/calcite/sql/babel/postgresql/SqlBegin.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql.babel.postgresql;
+
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlLiteral;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlNodeList;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.SqlSpecialOperator;
+import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.Symbolizable;
+import org.apache.calcite.sql.parser.SqlParserPos;
+import org.apache.calcite.sql.type.ReturnTypes;
+
+import com.google.common.collect.ImmutableList;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+import java.util.List;
+
+/**
+ * Parse tree node representing a {@code BEGIN} clause.
+ * @see <a href="https://www.postgresql.org/docs/current/sql-begin.html">BEGIN specification</a>
+ */
+public class SqlBegin extends SqlCall {
+  public static final SqlSpecialOperator OPERATOR =
```

Comment:
@zoudan It can not be private. Please check [parserPostgresqlImpls.ftl](https://github.com/apache/calcite/pull/2986/files#diff-5b1bdf395a9adcee120bea06e943ba76dd7c0dd3bc89bb2c871a5e1af24fe8fd):
```
return SqlBegin.OPERATOR.createCall(s.end(this), (SqlNode) transactionModeList)
```
 The same for all yours "do it too".